### PR TITLE
Add CONTRIBUTING_BASIC.md and CONTRIBUTING_ADVANCED.md

### LINKED ISSUE #2629: Split contributing.md into basic and advanced
Basic guide should go over basic contributing without setting up a local server. (working just in github). This will be good for absolute noobies who want to add languages or quotes. (Maybe include a link to a good youtube guide on git/github basics)

Advanced will go through the full process (what we have now)

### PR DESCRIPTION
**Pull request by Matthew-Kwan**
### Description

I've refactored the contribution guide into a basic and advanced version. 

`CONTRIBUTING.md` now has a section to help refer potential contributors to the right guide for the change they're trying to make. Most of the content has moved to the advanced contribution guide, but I've kept the guidelines here (open to suggestions on where these should go).

`CONTRIBUTING_BASIC.md` goes through how to use either the VS Code web editor or the Github Web UI to make commits directly through their browser, and open PRs.

`CONTRIBUTING_ADVANCED.md` is mostly a duplicate of the previous state of `CONTRIBUTING.md`, minus sections that stayed in the original file, and minus sections that are no longer relevant. 

Closes #2629 

https://github.com/monkeytypegame/monkeytype/issues/2629

### CODE REVIEW

**Comment by Miodec** [comment]
image

"For most cases, select the first option for now" but the screenshot shows second option which creates a pull request. Doesnt make sense if they want to commit multiple times or follow the guide because there is a section on creating a pull request just below that.

**Comment by Matthew-Kwan** [comment]
Good catch, I'll update the screenshot

**Review by sillyangel — DISMISSED** [review]
It better and it Clean up some things

**Comment by sillyangel** [comment]
i like it

**Comment by Matthew-Kwan** [comment]
@Miodec updated the screenshot to reflect the recommended radio option and added some text around that section to clarify it a bit further.

File: CONTRIBUTING_ADVANCED.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  - [NodeJS and NPM](#nodejs-and-npm)
+  - [Firebase](#firebase)
+  - [Mongo Setup](#mongo-setup)
+- [Building and Running Monkeytype](#building-and-running-monkeytype)
+- [Standards and Guidelines](#standards-and-guidelines)
+- [Questions](#questions)
+
+## Getting Started
+
+When contributing to Monkeytype, it's good to know our best practices, tips, and tricks. First, Monkeytype is written in ~~JavaScript~~ TypeScript, CSS, and HTML (in order of language usage within the project); thus, we assume you are comfortable in these languages or have basic knowledge of them. Our backend is in NodeJS and we use MongoDB to store our user data. Firebase is used for authentication. Furthermore, we use Prettier to format our code.
```
Comment (Miodec) [inline CONTRIBUTING_ADVANCED.md:18]: I think we can move this to contributing.md

File: CONTRIBUTING_ADVANCED.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Prerequisites
+
+While most contributions don't require that you install dependencies, there are a few tools you will need to be able to run the project (this is useful and almost always necessary for tasks like creating features and fixing bugs; running the project is also useful if you are contributing a theme and want to view it on the site before you contribute it). You will need a computer with a stable internet connection, a text editor, Git, Firebase, and NodeJS with version 16.13.2.
```
Comment (Miodec) [inline CONTRIBUTING_ADVANCED.md:22]: This is the advanced guide so some of the things here don't exactly apply.

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by Matthew-Kwan** [comment]
@Miodec moved `Getting Started` to `CONTRIBUTING.md` and adjusted the `Prerequisites` section in the advanced guide to tailor it more to the context.

**Comment by Miodec** [comment]
Thanks

**Comment by Matthew-Kwan** [comment]
Thanks for the review

**Pull request merged**